src/PS1/runtime/MeshReconstructor.cpp

```
@@ -20,16 +20,18 @@ uint32_t packDiffuse(uint8_t r, uint8_t g, uint8_t b)
+    return QStringLiteral("PS1Rip_tpage_%1_clut_%2_st%3")
         .arg(tpage, 4, 16, QChar('0'))
-        .arg(clut, 4, 16, QChar('0'));
+        .arg(clut, 4, 16, QChar('0'))
+        .arg(semiTrans & 3);
 }
 
-quint64 textureGroupKey(uint16_t tpage, uint16_t clut)
+quint64 textureGroupKey(uint16_t tpage, uint16_t clut, uint8_t semiTrans)
 {
-    return (static_cast<quint64>(tpage) << 32) | clut;
+    return (static_cast<quint64>(tpage) << 32) | clut
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_DECLARE_METATYPE is a macro then please configure it.

(unknownMacro)